src/quickgui/plugin/qgsquickmapcanvas.qml

```diff
@@ -343,4 +400,8 @@ Item {
       userInteractedWithMap()
     }
   }
+
+  QgsQuick.QgsQuickUtils {
```

Comment:
IMHO, I'd go down the road of a singleton right away, it's really meant to be used in other places too and making it a singleton from the get go will be more inviting for others to append useful stuff there.

Reply:
Makes sense, it is now a singleton class